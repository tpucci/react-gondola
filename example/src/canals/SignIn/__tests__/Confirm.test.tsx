import React from 'react';
import TestRenderer from 'react-test-renderer';

import { Confirm } from '../Confirm';

describe('Confirm', () => {
  it('renders correctly', () => {
    const testRenderer = TestRenderer.create(<Confirm />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
