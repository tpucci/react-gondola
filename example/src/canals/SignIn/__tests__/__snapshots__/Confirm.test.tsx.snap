// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm renders correctly 1`] = `
<View
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFAAAA",
          "flex": 1,
          "justifyContent": "space-around",
        }
      }
    >
      <Text>
        What is your first name ?
      </Text>
      <TextInput
        allowFontScaling={true}
        placeholder="Type here"
        rejectResponderTermination={true}
        underlineColorAndroid="transparent"
      />
      <View
        accessibilityRole="button"
        accessibilityStates={Array []}
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "padding": 8,
                  "textAlign": "center",
                },
              ]
            }
          >
            Next
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    collapsable={false}
    pointerEvents="none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
  <View
    collapsable={false}
    pointerEvents="none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
